### ISSUE TITLE
shortening breaks when html is copied/pasted from an area that has been shortened already

### ISSUE THREAD

**Issue by propertunist** [viralpatel/jquery.shorten#32]
thanks for this plugin. i have been using it on my site for a while and generally it is fine. i just found a problem though, that i have no idea of the cause of. someone posted a comment to my site that was partially freshly typed text and partially pasted text from another comment she had written. the shorten code correctly shortened the text, but when i click the 'expand' link, the full text is not revealed. the animation is triggered, but there is no more text to read...

edit: i have debugged this a bit now and found that the cause of the problem includes that the text that was pasted into the comment that i am applying the shortening function to... contains a class that had been previously added by the shorten function when it was applied to the source comment.
in other words, the text that the shorten function is being run against already contains an html 'class' definition called 'allcontent' because the user copied and pasted text from an area that already had been shortened previously on the page...

i changed the line:
```
$this.find(".allcontent").hide(); // Hide all text
```
to
```
$this.find(".allcontent:first-of-type").hide(); // Hide all text
```
to see if that might improve the situation but it didn't provide a quick workaround and instead some of the text was duplicated on the page.

**Comment by propertunist** [comment]
i have fixed this on my site now by filtering out the class 'allcontent' from all html before it reaches the browser.
